iree/compiler/Dialect/VM/Conversion/VMToEmitC/ConvertVMToEmitC.cpp

```diff
@@ -205,6 +235,29 @@ class ConvertVMToEmitCPass
     // support for control flow ops has landed in the c module target
     target.addIllegalOp<IREE::VM::CheckEQOp>();
 
+    // ExtI64: Constants
```

Comment:
can you just addIllegalDialect<IREE::VM::VMDialect>? or are you relying on some vm ops surviving into emitc?

Reply:
That should only be vm.func and vm.module. The other ops are unsupported at the moment and we can fail here.

Reply:
At the moment, some ops require special care in the printer/are explicitly supported in the printer, e.g. `vm.module`, `vm.func`, `vm.return`... Yes, we should invert the logic here, but I would prefer to do address this in a followup.